packages/safe-core-sdk/src/ethereumLibs/EthersAdapter.ts

```diff
@@ -152,12 +153,24 @@ class EthersAdapter implements EthAdapter {
     return this.#provider.call(transaction)
   }
 
-  ensLookup(name: string): Promise<string | null> {
-    return this.#provider.resolveName(name)
+  async ensLookup(name: string): Promise<string> {
+    const address = await this.#provider.resolveName(name)
+
+    if (!address) {
+      throw new Error(ErrorCodes._100)
+    }
+
+    return address
   }
 
-  ensReverseLookup(address: string): Promise<string | null> {
-    return this.#provider.lookupAddress(address);
+  async ensReverseLookup(address: string): Promise<string> {
+    const name = await this.#provider.lookupAddress(address);
```

Comment:
This should be in a try-catch, like the web3 version. Otherwise it can throw its own error (e.g. network down).